programs/vote/src/vote_state/mod.rs

```diff
@@ -31,7 +31,7 @@ pub const MAX_LOCKOUT_HISTORY: usize = 31;
 pub const INITIAL_LOCKOUT: usize = 2;
 
 // Maximum number of credits history to keep around
-const MAX_EPOCH_CREDITS_HISTORY: usize = 64;
+pub const MAX_EPOCH_CREDITS_HISTORY: usize = 64;
```

Comment:
Make this const `pub` shouldn't create much burden for us for maintainability or compatibility as this is very sensitive to the abi. So, we won't change this in near future.